homeassistant/components/myio/__init__.py

```diff
@@ -0,0 +1,143 @@
+"""The myIO integration."""
+from datetime import timedelta
+import logging
+
+from myio.comms_thread import CommsThread  # pylint: disable=import-error
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+)
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+from homeassistant.util import slugify
+
+from .const import CONF_PORT_APP, CONF_REFRESH_TIME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["sensor"]
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
Stale code? This config schema isn't used so it seems.

Reply:
CONFIG_SCHEMA deleted...